.drone.star

```diff
@@ -25,20 +30,10 @@ config = {
     },
     "phpunit": {
         "allDatabases": {
-            "phpVersions": [
-                "7.3",
-            ],
-        },
-        "reducedDatabases": {
             "phpVersions": [
                 "7.4",
             ],
-            "databases": [
-                "mariadb:10.2",
-            ],
-            "coverage": False,
         },
-    },
```

Comment:
https://drone.owncloud.com/owncloud/calendar/893
`.drone.star:39:4: got def, want '}' `

It looks like the script removed too many lines.